codex-ui/src/vue/components/alert/AlertTransition.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <transition-group
+    tag="div"
+    name="fade"
+    @leave="leave"
+  >
+    <slot />
+  </transition-group>
+</template>
+
+<script setup lang="ts">
+
+const leave = (el: unknown) => {
+  if (el instanceof HTMLElement) {
+    el.style.left = el.offsetLeft + 'px';
+    el.style.top = el.offsetTop + 'px';
+    el.style.width = getComputedStyle(el).width;
+    el.style.position = 'absolute';
+  }
+};
+
+</script>
+
+<style lang="postcss">
+.fade-enter-active,
+.fade-leave-active {
+  transition: opacity 0.5s ease, transform 0.5s ease;
+}
+
+.fade-enter-from,
+.fade-leave-to {
+  opacity: 0;
+  transform: translateY(30px);
```

Comment:
lets use simple fade-out effect for leaving

```suggestion
```

Reply:
please look at the updated one, if its good enough